src/styles/variable.scss

```diff
@@ -0,0 +1,12 @@
+$block-bg: #eee;
+$day-size: 100px;
+$block-gap: 1px;
+$calendar-padding: 10px;
+$count-columns: 7;
+$days: mon 0,
+  tue calc(#{$day-size} + $block-gap),
+  wed calc((#{$day-size * 2}) + ($block-gap * 2)),
+  thu calc((#{$day-size * 3}) + ($block-gap * 3)),
+  fri calc((#{$day-size * 4}) + ($block-gap * 4)),
+  sat calc((#{$day-size * 5}) + ($block-gap * 5)),
+  sun calc((#{$day-size * 6}) + ($block-gap * 6));
```

Comment:
you should calculate this values at `@each $day, $margin-left in $days`
here you need only a day number